woocommerce/api/class-sv-wc-api-base.php

```diff
@@ -263,10 +280,11 @@ protected function broadcast_request() {
 		);
 
 		$response_data = array(
-			'code'    => $this->get_response_code(),
-			'message' => $this->get_response_message(),
-			'headers' => $this->get_response_headers(),
-			'body'    => $this->get_sanitized_response_body() ? $this->get_sanitized_response_body() : $this->get_raw_response_body(),
+			'code'       => $this->get_response_code(),
+			'message'    => $this->get_response_message(),
+			'headers'    => $this->get_response_headers(),
+			'body'       => $this->get_sanitized_response_body() ? $this->get_sanitized_response_body() : $this->get_raw_response_body(),
+			'from_cache' => $this->is_response_loaded_from_cache(),
```

Comment:
@ChaseWiseman this produces log messages like the ones below:

When response was not loaded from cache:
```
body: { ... }
from_cache:
```

When response _was_ loaded from cache:
```
body: { ... }
from_cache: 1
```

I'm not particularly fond of the `1` and empty string representation here. It would be more readable if the logs would say `yes|no`, but that is harder to interpret programmatically. Would it make sense to add a formatter around this value [here](https://github.com/skyverge/wc-plugin-framework/blob/146ebc07aeaf8dc11378e51df1b0447cff83fefd/woocommerce/class-sv-wc-plugin.php#L721)?

Also, would it make sense to only add this key to the array if the request class supports caching?

Reply:
I've addressed the above in b3032d4 and e5272cd. Let me know what you think?